### PR TITLE
Refresh the commit access policy DOC 

### PR DESCRIPTION
**Pull request by sylvestre**
Just refreshing a bit the content
cc @globau

### CODE REVIEW

**Review by stevejalim — APPROVED** [review]
LGTM @sylvestre - shall I merge this?

**Comment by sylvestre** [comment]
@stevejalim I would like @globau to double check if you don't mind :)

File: bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html [hunk 1]
```diff
@@ -100,10 +100,12 @@ <h2 id="Level3">Level 3 - Core Product Access</h2>
...
+<p>This permission gives access to check into any tree from which executable code becomes part of our core products - Firefox, GeckoView and Thunderbird. This is fundamentally a statement of trust in and familiarity with an individual, and so access to one such tree gives access to all such trees, although social controls may prevent people checking in to certain ones.</p>
 
 <p>People with this access may not actually be working on Firefox, Firefox for Android or Thunderbird, but their ability to affect them means that the level of trust and familarity required for access are higher.</p>
```
Comment (globau) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html:105]: As you changed `Firefox for Android` to `GeckoView` in the previous point, the same replacement should happen here.

File: bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html [hunk 2]
```diff
@@ -100,10 +100,12 @@ <h2 id="Level3">Level 3 - Core Product Access</h2>
...
 
+<p>With this level (scm_level_3), the developer can use <a href="https://moz-conduit.readthedocs.io/en/latest/lando-user.html">lando</a> to land code.</p>
```
Comment (globau) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html:107]: `lando` → `Lando`

**Review by globau — CHANGES_REQUESTED** [review]

File: bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html (hunk 1 above)
Comment (stevejalim) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html:105]:
```suggestion
<p>People with this access may not actually be working on Firefox, GeckoView or Thunderbird, but their ability to affect them means that the level of trust and familarity required for access are higher.</p>
```

File: bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html [hunk 3]
```diff
@@ -100,9 +100,11 @@ <h2 id="Level3">Level 3 - Core Product Access</h2>
...
+<p>This permission gives access to check into any tree from which executable code becomes part of our core products - Firefox, GeckoView and Thunderbird. This is fundamentally a statement of trust in and familiarity with an individual, and so access to one such tree gives access to all such trees, although social controls may prevent people checking in to certain ones.</p>
 
-<p>People with this access may not actually be working on Firefox, Firefox for Android or Thunderbird, but their ability to affect them means that the level of trust and familarity required for access are higher.</p>
+<p>People with this access may not actually be working on Firefox, GeckoView or Thunderbird, but their ability to affect them means that the level of trust and familarity required for access are higher.</p>
+
+<p>With this level (scm_level_3), the developer can use <a href="https://moz-conduit.readthedocs.io/en/latest/lando-user.html">lando</a> to land code.</p>
```
Comment (stevejalim) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html:107]:
```suggestion
<p>With this level (scm_level_3), the developer can use <a href="https://moz-conduit.readthedocs.io/en/latest/lando-user.html">Lando</a> to land code.</p>
```

**Comment by stevejalim** [comment]
@sylvestre @globau I've amended this PR based on the comments from @globau

Shall I merge this to `main` now, or does it need further review?

**Review by globau — APPROVED** [review]
lgtm

**Pull request merged**